hw/ip/lc_ctrl/doc/_index.md

```diff
@@ -195,41 +200,220 @@ This ensures that during specific states (RAW, TEST_LOCKED, SCRAP, INVALID) it i
-During other states, it is not possible to either read or modify the collateral.
-This specifically limits the danger of rogue software images during any TEST_UNLOCKED state.
-
-However, as PROVISION_RD_EN and PROVISION_WR_EN only gate functional access and not DFT access, it is still possible for a malicious agent to bypass this protection by abusing scan shift/capture mechanics.
-
-Note also that PROVISION_RD_EN and PROVISION_WR_EN provide blanket control over all provision related collateral.
-Each collateral contains more specific OTP / flash / Software based decoding that further limits accessibility.
-See the flash and OTP sections for more details.
-
-### KEY_MANAGER_EN
+#### KEY_MANAGER_EN
 
 The KEY_MANAGER_EN signal allows the key manager to function normally.
 When this signal is logically disabled, any existing key manager collateral is uninstantiated and wiped; further instantiation and generation calls for the key manager are also made unavailable.
 
 The KEY_MANAGER_EN signal is active only during DEV / PROD / PROD_END / RMA.
 
-### CLK_BYP_REQ
+#### CLK_BYP_REQ
 
 The CLK_BYP_REQ signal switches the main system clock to an external clock signal in RAW and TEST_LOCKED states when initiating a life cycle transition.
 This is needed since the internal clock source may not be fully calibrated yet in those states, and the OTP macro requires a stable clock frequency in order to reliably program the fuse array.
 The CLK_BYP_REQ signal is only asserted when a transition command is issued.
 
-### ESCALATE_EN
+#### ESCALATE_EN
 
 The ESCALATE_EN signal is available in all life cycle states and is asserted if for any reason the alert subsystem decides to move the life cycle state into the ESCALATION state.
 
-## Control Signal Propagation
+### Life Cycle Access Control Signals
+
+The individual signals summarized in the table below are described in the following subsections.
+
+{{< snippet "hw/ip/lc_ctrl/doc/lc_ctrl_access_signals_table.md" >}}
+
+Signals marked with an asterisk (Y\*) are only asserted under certain conditions as explained in detail below.
+
+#### CREATOR_SEED_SW_RW_EN and OWNER_SEED_SW_RW_EN
+
+These signals control whether the non-volatile provisioning of life cycle related collateral can be accessed.
+The signals can only be active during DEV / PROD / PROD_END / RMA.
+During other states, it is not possible to either read or modify the collateral.
+This specifically limits the danger of rogue software images during any TEST_UNLOCKED state.
+However, as these signals only gate functional access and not DFT access, it is still possible for a malicious agent to bypass this protection by abusing scan shift/capture mechanics.
+
+While the OWNER_SEED_SW_RW_EN is statically enabled in the states shown above, the CREATOR_SEED_SW_RW_EN is only enabled if the device has not yet been personalized (i.e., the OTP partition holding the root key has not been locked down yet).
```

Comment:
so just one minor thing here.  The `creator_seed_sw_rw_en` is also used to control the provision of contents in flash. So this would mean that the user would need to..

1. configure otp contents
2. configure flash contents
3. now lock down otp (or swap steps 2 and 1), is that right? 

If so, we just need to make sure it is very clear that this has to be done in a specific order. 
I think this dependency is fine, we just have to be careful since it'll probably be easy for someone to do it wrong a few times and accidentally lock themselves out of configuring flash.

Let me know if I misunderstood.

Reply:
your understanding is correct - we basically consolidated `ID_STATE` with the `SECRET2_DIGEST`.

both provisioning orders you mentioned are fine (1 2 3 or 2 1 3).

where do you think would be the best place to capture this? do we have a provisioning sequence document somewhere?

Reply:
i think that will probably come from @moidx later.
I think now, could you put a "CAUTION" section in the programmers model? :) and just point out this dependency. 
It would be good if @cindychip if aware of this also when we create top level tests.

Reply:
ok sounds good, I'll add that :).